.github/workflows/golangci-lint.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         uses: golangci/golangci-lint-action@v3
         with:
           version: v1.50.1
-          only-new-issues: ${{github.event.schedule == ''}} # show only new issues, unless it's a scheduled run
+          only-new-issues: ${{ github.event.schedule != '' }} # show only new issues, unless it's a scheduled run
```

Comment:
Wasn't this correct before? It is opposite of the others because we don't want this constraint for scheduled runs.

Reply:
Thanks, yeah good catch. This was a remnant from testing.